Components/FlightControl/TimerTransitions.cpp

```diff
@@ -0,0 +1,44 @@
+/**
+ ******************************************************************************
+ * File Name          : TimerTransitions.cpp
+ * Description        : Primary Watchdog task, default task for the system.
+ ******************************************************************************
+*/
+#include "GPIO.hpp"
+#include "SystemDefines.hpp"
+#include "Timer.hpp"
+#include "TimerTransitions.hpp"
+#include "FlightTask.hpp"
+
+TimerTransitions::TimerTransitions() {
+	ignitionCountdown = new Timer(IngnitionToLaunchCallback);
+	ignitionCountdown->ChangePeriodMs(IGINITION_TIMER_PERIOD);
+}
+
+void TimerTransitions::IgnitionSequence() {
+//    SOAR_PRINT("Entering IGNITION state...\n");
```

Comment:
Let's get rid of these commented out SOAR_PRINTs

Reply:
Can i keep them in for now and when i am done all transitions then i will reomve them

Reply:
Yep